### PR TITLE
Default coc-settings.json file

### PR DESCRIPTION
**Pull request by nwessing**
Vista plugin for visualizing document symbols
Colors for CoC virtual text

### CODE REVIEW

**Comment by panzacoder** [comment]
Can we merge this in @jondkinney @nwessing? I think this is needed for auto-formatting to work without Ale

**Comment by noahsettersten** [comment]
I've approved the PR; somehow I missed it earlier when it came through.

Good call on Vista as well. That's been a great switch from relying on ctags.

File: coc-settings.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "diagnostic.virtualText": true,
+  "codeLens.enable": true,
+  "eslint.autoFixOnSave": true,
```
Comment (panzacoder) [inline coc-settings.json:4]: I just tried adding this in locally and didn't have success. I did however get autoformatting working with these settings: 
```"coc.preferences.formatOnType": true,
  "coc.preferences.formatOnSaveFiletypes": [
    "javascript",
    "json",
    "elixir",
    "ruby",
    "css",
    "ts",
    "jsx",
    "tsx",
    "markdown"
  ], ```

Is there something else that needs to be updated for this to work?

Comment (nwessing) [reply coc-settings.json:4]: Good catch, eslint has its own flag for some reason. But you definitely need to add these for elixir and the other to auto format.

**Comment by nwessing** [comment]
> I've approved the PR; somehow I missed it earlier when it came through.
> 
> Good call on Vista as well. That's been a great switch from relying on ctags.

Thanks! I thought it might be useful to view all symbols in a buffer.

**Pull request merged**